js/archive.js

```diff
@@ -0,0 +1,27 @@
+/* Obly run this if we are online*/
```

Comment:
Wee typo

Reply:
That's in every branch so far, why fix it now ;)